.github/CONTRIBUTING.md

```diff
@@ -42,21 +42,32 @@ Gradle uses pull requests for contributions. Fork [gradle/gradle](https://github
     git config user.name 'First Last'
     git config user.email user@example.com
 
+### IntelliJ
+
 You can generate the IntelliJ project by running
 
-    ./gradlew idea
+    gradlew idea
+
+### Eclipse
 
-You can generate the Eclipse projects by running
+You can generate the Eclipse project <ith the following steps
+  
+1. Install Eclipse 4.5 (Mars) at least
+2. Install the Groovy Eclipse plugin from http://dist.springsource.org/snapshot/GRECLIPSE/e4.5/
+3. Make sure you have a Java 8 compatible JDK configured in your workspace
+4. In `Window->Preferences->Groovy->Compiler`, check `Enable Script folder support` and add `**/*.gradle`
+5. Run `gradlew eclipse` from the root directory
+6. Import all projects using the "Import Existing Projects into Workspace" wizard
 
-    ./gradlew eclipse
 
+ 
 ### Code Change Guidelines
 
 All code contributions should contain the following:
 
 * Unit Tests (using [Spock](http://spockframework.org/spock/docs/1.1-rc-2/index.html)) for any logic introduced
 * Integration Test coverage of the bug/feature at the level of build execution. Please annotate tests guarding against a specific GitHub issue `@Issue("gradle/gradle#123")`.
-* Documentation in the User Guide and DSL Reference (under `subprojects/docs/src/docs`). You can generate docs by running `./gradlew :docs:docs`.
+* Documentation in the User Guide and DSL Reference (under `subprojects/docs/src/docs`). You can generate docs by running `gradlew :docs:docs`.
```

Comment:
Please don't remove the `./` as this is needed in some environments. Same below.